public/oldIndex.html

```diff
@@ -224,8 +224,13 @@ <h2>Stop waiting.<br>Start Getting Involved.</h2>
             Email: <a href="mailto:info@helpingfamiliesinsv.org">info@helpingfamiliesinsv.org</a>
         </p>
         <ul class="list-inline list-social">
+            <li class="list-inline-item ">
+                <a href="https://www.facebook.com/Helping-Families-in-Silicon-Valley-102948694690106?helping=families">
+                    <i class="fab fa-facebook-f"></i>
+                </a>
+            </li>
             <li class="list-inline-item social-instagram">
-                <a href="https://instagram.com/helpingfamiliesinsv">
+                <a href="https://instagram.com/helpingfamiliesinsiliconvalley?helping=families">
```

Comment:
lets add new links to react components . I think there is an object for this

Reply:
ok

Reply:
done